src/Evaluator/PositionEvaluator.php

```diff
@@ -44,7 +44,7 @@ private function getPieceValue(int $index, ?array $piece): int
             case 'q':
                 return $isWhite ? PositionValue::QUEEN_WHITE[$i][$j] : PositionValue::QUEEN_BLACK[$i][$j];
             case 'k':
-                return $isWhite ? PositionValue::KNIGHT_WHITE[$i][$j] : PositionValue::KING_BLACK[$i][$j];
+                return $isWhite ? PositionValue::KING_WHITE[$i][$j] : PositionValue::KING_BLACK[$i][$j];
```

Comment:
:scream: it looks like I'm an amateur :wink:

Reply:
Just a typo ;)